__tests__/hooks/useFeedHeader.test.js

```diff
@@ -54,7 +54,7 @@ describe('Feed Header should run correctly', () => {
     act(() => {
       result.current.onBackNormalUser();
     });
-    expect(mockedNavigate).toHaveBeenCalled();
+    expect(mockedGoBack).toHaveBeenCalled();
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change from `mockedNavigate` to `mockedGoBack` seems appropriate if the intention is to go back to the previous screen rather than navigate to a new one. However, it's important to ensure that this change doesn't affect other parts of the application where a navigation to a new screen might be expected.
```diff
-    expect(mockedNavigate).toHaveBeenCalled();
+    expect(mockedGoBack).toHaveBeenCalled();
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->